setup.py

```diff
@@ -68,6 +68,8 @@
           'dev': all_extra,
           'plot': plot,
           'all': all_extra,
+          'jupyter': [],
```

Comment:
this will not do anything ...
yes, it allows running `pip install -e .[jupyter]` - but won't install jupyter.
The correct syntax  is `<key>:[<additional packages to install>]`

```
'jupyter': ['jupyter'],
```

I'm not sure if "key" (the left side) should be replaced by "analysis" or something similar (`pip install freqtrade[plot,analysis]` vs. `pip install freqtrade[plot,jupyter]`) ... but i'll leave this up to you

Reply:
Thanks for explaining. I see how that works. I added a section for useful packages for jupyter and keyed with 'jupyter'. I also added it to the all_extras list.